.github/workflows/release.yml

```
@@ -140,13 +140,22 @@ jobs:
       - uses: actions/setup-python@v3
         with:
           python-version: ${{ matrix.python-version }}
+      - uses: google-github-actions/auth@v0
+        continue-on-error: true
+        with:
+          credentials_json: ${{ secrets.gcs_bazel_cache }}
       - name: Build macOS wheels
         run: |
           python --version
           python -m pip install delocate wheel setuptools numpy six --no-cache-dir
 
           ./configure.py
 
+          if [[ -n $GOOGLE_APPLICATION_CREDENTIALS ]]; then
+            echo -e 'build --remote_http_cache=https://storage.googleapis.com/plumerai-bazel-cache/lce-release-macos' >> .bazelrc.user
```

Just a question, would it help for Bazel to do something like this instead:
```suggestion
            echo -e 'build --remote_http_cache=https://storage.googleapis.com/plumerai-bazel-cache/lce-release-macos-${{ matrix.python-version }}' >> .bazelrc.user
```
Then each Python version will have its own cache. Or is Bazel smart enough to not overwrite those caches?

---

Good idea, I'll add that.